src/main/java/database/sql/ddl/EntityClassInspector.java

```diff
@@ -0,0 +1,82 @@
+package database.sql.ddl;
+
+import jakarta.persistence.*;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class EntityClassInspector {
+    private final Class<?> entityClass;
+
+    public EntityClassInspector(Class<?> entityClass) {
+        this.entityClass = entityClass;
+    }
+
+    public String getTableName() {
+        Table tableAnnotation = entityClass.getAnnotation(Table.class);
+        if (tableAnnotation != null && !tableAnnotation.name().isEmpty()) {
+            return tableAnnotation.name();
+        } else {
+            return entityClass.getSimpleName();
+        }
```

Comment:
```suggestion
        if (tableAnnotation != null && !tableAnnotation.name().isEmpty()) {
            return tableAnnotation.name();
        }
        return entityClass.getSimpleName();
```
```
객체지향 생활 체조 원칙
규칙 2: else 예약어를 쓰지 않는다.
```

else를 사용하지 않고 바로 리턴할 수 있겠네요.